marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/LoadUserArtifactsCommand.java

```diff
@@ -155,9 +159,10 @@ private void loadModels(HubClient hubClient) throws IOException {
                 public FileVisitResult preVisitDirectory(Path dir, BasicFileAttributes attrs)  {
                     logger.info("Loading models from directory " + dir);
                     modulesFinder.findModules(dir.toString()).getAssets().forEach(r -> {
-                        try {
+                        try (InputStream inputStream = r.getInputStream()) {
```

Comment:
We've got 5 instances of code all wanting to do the same thing - convert a File or Resource into a JsonNode, and now they all want to replace tokens at the same time. 

It's best to do this via a new private method - particularly as when we add a 6th instance of the code, we can just reuse that method and not worry about forgetting to replace tokens. 

Since we know that everything is a File here, the 2 places that use a Resource should just do "resource.getFile()". The reusable method can then have a signature of:

    private JsonNode readArtifact(File file);

And it should catch any exception that occurs and throwing a generic error message of:

    throw new RuntimeException("Unless to read file " + f.getFilename() " + as JSON; cause: " + e.getMessage(), e);